guiscrcpy/cli.py

```diff
@@ -133,6 +133,13 @@ def cli(
         # it will log the traceback to the terminal
         # but it would not be visible to users without CLI interface
         # enable High DPI scaling
+
+        if platform.system() == "Windows":
+            import ctypes
+
+            appid = "srevinsaju.guiscrcpy"
+            ctypes.windll.shell32.SetCurrentProcessExplicitAppUserModelID(appid)
```

Comment:
It would be nice to have a inline comment explaining that these lines are meant for what, etc.

Reply:
You can see this https://stackoverflow.com/a/1552105/9986755